samples/android/app/src/androidTest/java/io/github/ivsivak/android_kohttp/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,22 @@
+package io.github.ivsivak.android_kohttp
```

Comment:
not sure if it's a good idea to have a different package root.
#128 will move everything to `io.github.rybalkinsd.kohttp.<subproject-name>` path